composer.json

```diff
@@ -54,7 +54,10 @@
         "friends-of-behat/mink-browserkit-driver": "^1.6",
         "friends-of-behat/mink-extension": "^2.7",
         "friends-of-behat/symfony-extension": "^2.0",
+        "phpunit/phpunit": "^9.5",
```

Comment:
See the configuration of testing in Symfony Docs: https://symfony.com/doc/5.x/testing.html

They recommend to install `symfony/test-pack` which also installes `phpunit/phpunit` and `symfony/css-selector`